commands/cluster-slots.md

```diff
@@ -34,8 +34,10 @@ Each nested result is:
   - ...continues until all replicas for this master are returned.
 
 Each result includes all active replicas of the master instance
-for the listed slot range.  Failed replicas are not returned.
+for the listed slot range. Failed replicas are not returned.
 
+The command response is deterministic across all nodes in a cluster, which means that if two nodes return the same response they have the same view of the cluster.
+Primaries are ordered by the slots they serve and then replicas are ordered lexicographically by the node-id they were assigned by the cluster. 
 The third nested reply is guaranteed to be the networking information of the master instance for the slot range.
 All networking information after the third nested reply are replicas of the master.
 
```

Comment:
Here, we're jumping between describing the primary, the replicas and the slot ranges.

This section is about a *slot range*, which is each line in the nested reply. Looking at the whole text from "Each result includes all active replicas of the master instance for the listed slot range" until the example below, I would start with describing the ordering of the slot ranges (including the sentence that explains that if a shard has non-contiguous slots, then the networking information is repeated for each range) and then say what the primary for this slot range is and that the replicas for this slot range are ordered lexicographically by node-id.

Rough idea:

> Each result includes all active replicas of the primary instance for the listed slot range.
> Failing replicas are not returned.
>
> The slot ranges are returned in ascending order.
> If a cluster shard serves non-contiguous slots (e.g. 1-400,900,1800-6000) then primary and replica networking information results are duplicated for each slot range.
>
> The third nested reply is the networking information of the primary instance for the slot range.
> All networking information after the third nested reply are replicas of the primary.
> The replicas of each primary are ordered lexicographically by the node-id they were assigned by the cluster.